src/lib/plugin-store.js

```diff
@@ -1,12 +1,30 @@
 'use strict'
 
 const Sequelize = require('sequelize')
-const Database = require('five-bells-shared').DB(Sequelize)
+const log = require('../common/log').create('plugin-store')
+const url = require('url')
 
 class PluginStore {
   constructor (uri, name) {
     this.name = name
-    this.db = new Database(uri)
+    const options = {
+      logging: log.debug,
+      omitNull: true,
+      // All transactions should be done with isolation level SERIALIZABLE
+      // TOOD:
```

Comment:
What's this TODO for?

Reply:
This "TOOD" was going to say: So we still need SERIALIZABLE isolation level. (My guess is no.)